docs/handbook/getting-started.md

```diff
@@ -582,13 +580,13 @@ Replace `X` with the appropriate letter for your USB device.
 #### Method 1 (New releases)
 
 ```
-sudo dd bs=4M if=./image.usb of=/dev/sdb status=progress && sync
+sudo /usr/gnu/bin/dd bs=4M if=./image.usb of=/dev/sdb status=progress && sync
 ```
 
 <!-- NOTE: --> <i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **NOTE:**
 <div class="well">
-Recent releases of OpenIndiana Hipster use the GNU version of dd (/usr/gnu/bin/dd), which should support the `status=progress` option.
-Older releases however may still be using illumos dd (/usr/bin/dd) which may not support this option.
+The command above uses the GNU version of dd `/usr/gnu/bin/dd`, which should support the `status=progress` option.
+If however you are using illumos dd `/usr/bin/dd`, bear in mind it may not support this option.
```

Comment:
I you use illumos dd, command will look like sudo /usr/bin/dd if=./image.usb of=/dev/rdsk/c3t2d0p0 bs=4194304